.github/workflows/install_deps.py

```diff
@@ -39,7 +39,7 @@ def install_packages(what):
     }
     packages['exhibitor'] = packages['zookeeper']
     packages = packages.get(what, [])
-    ver = str({'etcd': '9.6', 'etcd3': '9.6', 'consul': 10, 'exhibitor': 11, 'kubernetes': 12, 'raft': 13}.get(what))
+    ver = str({'etcd': '9.6', 'etcd3': '13', 'consul': 12, 'exhibitor': 11, 'kubernetes': 13, 'raft': 12}.get(what))
```

Comment:
There is no downgrade. 12 is the postgres version and it is orthogonal to the type of DCS.

The main point here is that we want to test not only against different DCS but also with different postgres versions.
Since `pg_replication_slot_advance()` is only available starting from postgres 11 we run tests on with 11, 12, and 13 and distribute them across all possible DCS.

In order to make sure that there is no degradation with older postgres versions we still run one behave test with 9.6 against etcd.